packages/components/src/tooltip/test/index.js

```diff
@@ -75,18 +70,23 @@ describe( 'Tooltip', () => {
 				</Tooltip>
 			);
 
-			const button = wrapper.find( 'button' );
-			button.simulate( 'focus', event );
+			const button = screen.getByRole( 'button' );
+			button.focus();
 
-			const popover = wrapper.find( 'ForwardRef(Popover)' );
-			expect( originalFocus ).toHaveBeenCalledWith( event );
+			const popover =
+				container.getElementsByClassName( 'components-popover' );
+			expect( originalFocus ).toHaveBeenCalledWith(
+				expect.objectContaining( {
+					type: 'focus',
+				} )
+			);
 			expect( popover ).toHaveLength( 1 );
 		} );
 
-		it( 'should show not popover on focus as result of mousedown', async () => {
+		it( 'should not show popover on focus as result of mousedown', async () => {
```

Comment:
I'm not quite understanding what is this trying to test — a mousedown event, followed by focus, followed by mouseup? Can we just replace it with a `click` event from `@testing-library/user-event` ?

Reply:
Fixed by 10274ad

The original implementation was to wait for the time indicated by the constant (`TOOLTIP_DELAY`) and then check for the popover to appear.
I simply wait for the popover text to appear by the `findBy***` method.
Should I strictly wait the time specified in `TOOLTIP_DELAY`?

Reply:
Fixed by d5e17de.
This test confirmed that the tooltip doesn't appear even after the time specified in TOOLTIP_DELAY has elapsed since the button was clicked.

Reply:
Reactored by dda605a
See: https://github.com/WordPress/gutenberg/pull/43061#discussion_r949384687